src/dug/core/parsers/_base.py

```diff
@@ -29,7 +29,18 @@ def add_concept(self, concept):
         self.concepts[concept.id] = concept
 
     def jsonable(self):
-        return self.__dict__
+        """Output a pickleable object
+
+        used by dug.utils.complex_handler. Because search_terms and
+        optional_terms are considered unsorted lists by the parsers but will be
+        treated as sorted lists by python, sorting the lists before output
+        prevents changes in ordering from being treated as a change in output by
+        incremental change detection.
+        """
+        outdict = self.__dict__
```

Comment:
can we avoid exposing __dict__ by returning it? it supposed to be private

Reply:
returning self.__dict__ was in the code prior to this change. Do you think we should be making a copy before returning it?